server/routes/comment.js

```diff
@@ -0,0 +1,19 @@
+import express from 'express';
+import Comments from '../controllers/Comments';
+import middlewares from '../middlewares';
+
+const route = express.Router();
+
+const { verifyToken, validateCommentBody, getSessionFromToken } = middlewares;
+
+route.post(
+  '/:slug/comments',
+  verifyToken,
+  getSessionFromToken,
+  validateCommentBody,
```

Comment:
Add `checkUserVerification` middleware to this route

Reply:
Added